crypto/ct/ct_x509v3.c

```diff
@@ -30,12 +30,50 @@ static int i2r_SCT_LIST(X509V3_EXT_METHOD *method, STACK_OF(SCT) *sct_list,
     return 1;
 }
 
+static int set_sct_list_source(STACK_OF(SCT) *s, sct_source_t source) {
+    if (s != NULL) {
+        int i;
+        for (i = 0; i < sk_SCT_num(s); i++) {
+            int res = SCT_set_source(
+                sk_SCT_value(s, i), SCT_SOURCE_X509V3_EXTENSION);
```

Comment:
I'm not sure if it coding style can be interpreted like that. But since you have to fix some styling maybe this should rather look like this
```
           int res = SCT_set_source(sk_SCT_value(s, i),
                                                    SCT_SOURCE_X509V3_EXTENSION);
```